infra/build/functions/ci.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2022 Google LLC
```

Comment:
`functions` seems like the wrong dir for this -- it's intended for code that's deployed onto GCF, but this wouldn't be right?